docs/en/Tiered-Storage-on-Alluxio.md

```diff
@@ -1,88 +1,117 @@
+between the two by enabling and disabling the space reserver which handles asynchronous eviction.
+For example, to turn off asynchronous eviction:
+
+```
+alluxio.worker.tieredstore.reserver.enabled=false
+```
+
+Asynchronous eviction relies on a periodic space reserver thread in each worker to evict data. It
+waits until the worker storage utilization reaches a configurable high watermark. Then it evicts
+data based on the eviction policy until it reaches the configurable low watermark. For example, if
+we had the same 16+100+100=216GB storage configured, we can set eviction to kick in at around 200GB
+and stop at around 160GB:
+
+```
+alluxio.worker.tieredstore.level0.watermark.high.ratio=0.9
+alluxio.worker.tieredstore.level0.watermark.low.ratio=0.75
+```
 
-#### Pinning Files
+Synchronous eviction is the legacy implementation of eviction. It waits for a client to request more
+space than is currently available on the worker and then kicks off the eviction process to free up
+enough space to serve that request. This leads to many small eviction attempts, which is less
+efficient.
 
-A user can control the placement and movement of their files is using *pin* and *unpin*
-files. When a file is pinned, its blocks will not be evicted. However, users can still promote
-blocks of pinned files to move blocks to the top tier.
+Two sub-components of the eviction process are the evictor and allocator. These can be customized to
+achieve various desired data management policies.
 
-An example of how to pin a file:
+### Evictors
 
-{% include Tiered-Storage-on-Alluxio/pin-file.md %}
+Alluxio uses evictors for deciding which blocks to evict, when space needs to be
+freed. Alluxio supports custom evictors, and implementations include:
 
-Similarly, the file can be unpinned through:
+* **GreedyEvictor**
 
-{% include Tiered-Storage-on-Alluxio/unpin-file.md %}
+    Evicts arbitrary blocks until the required size is freed.
 
-Since blocks of pinned files are no longer candidates for eviction, clients should make sure to
-unpin files when appropriate.
+* **LRUEvictor**
+
+    Evicts the least-recently-used blocks until the required size is freed.
+
+* **LRFUEvictor**
+
+    Evicts blocks based on least-recently-used and least-frequently-used with a configurable weight.
+    If the weight is completely biased toward least-recently-used, the behavior will be the same as
+    the LRUEvictor.
+
+* **PartialLRUEvictor**
+
+    Evicts based on least-recently-used but will choose StorageDir with maximum free space and
+    only evict from that StorageDir.
+
+In the future, additional evictors will be available. Since Alluxio supports custom evictors,
+you can also develop your own evictor appropriate for your workload.
+
+When using synchronous eviction, it is recommended to use small block size (around 64MB),
+to reduce the latency of block eviction. When using the [space reserver](#space-reserver), block
+size does not affect eviction latency.
 
 ### Allocators
```

Comment:
do you think we should hide allocator section here?
I don't think it is recommended to change this setting.

Reply:
I've removed the allocator section